src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -494,6 +500,25 @@ private Task UpdateStatusCommentAsync(IRemote darc, string prUrl, string message
             }
         }
 
+        private async Task AddDependencyFlowEventsAsync(
+            IEnumerable<SubscriptionPullRequestUpdate> subscriptionPullRequestUpdates, 
+            DependencyFlowEventType flowEvent, 
+            DependencyFlowEventReason reason, 
+            MergePolicyCheckResult policy)
+        {
+            string updateReason = reason == DependencyFlowEventReason.New || 
+                                  reason == DependencyFlowEventReason.AutomaticallyMerged ? 
+                                  $"{reason}" : $"{reason}{policy}";
+            foreach (SubscriptionPullRequestUpdate update in subscriptionPullRequestUpdates)
+            {
+                ISubscriptionActor actor = SubscriptionActorFactory(new ActorId(update.SubscriptionId));
+                if (!await actor.AddDependencyFlowEventAsync($"{flowEvent}", updateReason, "PR"))
```

Comment:
Is this just a roundabout way of calling `flowEvent.ToString()`? I feel like I'd rather see the ToString call, since it's always weird to see `$"{thing}"`, since it looks like a no op, but is in fact hiding a very important translation later.